src/main/java/appeng/integration/IntegrationType.java

```diff
@@ -62,7 +63,7 @@ public enum IntegrationType {
 
     Chisel(IntegrationSide.BOTH, "Chisel", "chisel"),
 
-    Jabba(IntegrationSide.BOTH, "Jabba", "JABBA");
+    Jabba(IntegrationSide.BOTH, "Jabba", "JABBA"),;
```

Comment:
nit: unnecessary comma, I guess you tried to put semicolon in new line but spotless declined...

Reply:
spotless has been getting on my nerves lately for debugging/hotswap, i disable it for dev and reenable only at the end so i missed this